assets/lokomotive-kubernetes/bare-metal/flatcar-linux/kubernetes/profiles.tf

```diff
@@ -224,6 +224,7 @@ data "template_file" "worker-configs" {
     cluster_dns_service_ip = module.bootkube.cluster_dns_service_ip
     cluster_domain_suffix  = var.cluster_domain_suffix
     ssh_keys               = jsonencode(var.ssh_keys)
+    kubelet_labels         = join(",", concat(["node.kubernetes.io/node"], var.labels))
```

Comment:
And we can remove the `join()` and `concat()` if we do that on on worker.yaml.tmpl and we use labels as a string (as we do on packet).

Why not do the same as on Packet, for example?

Reply:
Hmm, ideally `join()` should be called inside the function, but that does not work, because we use `template_file` instead of `templatefile()`. Let me try fixing it.

Reply:
I pushed the commit to change it to use `template_file` + `null_resource`.

Reply:
That seems overly complicated for no good reason, IMHO. It seems more readable as it is here.

Or isn't it easier to move it to `tempaltefile()` instead to add a null resource? Would that solve the join thing?

Reply:
Hm, I messed something up. I thought the output of this template is used in more than one place, but this is the case only for `container-linux-install-configs`, which I decided not to touch in this PR.

I removed the `null_resource` and I use `templatefile()` directly.

Also, for future reference, if we need to use `count.index` with `templatefile`, `for` loop with `locals` will do just fine.